### PR TITLE
Refactor custom fee for contract call

### PR DESCRIPTION
**Pull request by mustafauzunn**
**Description**:

Refactor custom fee for contract call

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by IvanKavaldzhiev — DISMISSED** [review]
LGTM!

File: hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/token/TokenAccessorImpl.java [hunk 1]
```diff
@@ -83,6 +85,9 @@
 @RequiredArgsConstructor
 public class TokenAccessorImpl implements TokenAccessor {
 
+    public static final String SELECT_QUERY ="select * from custom_fee  where token_id = :tokenId and created_timestamp = (select created_timestamp from custom_fee  where token_id = :tokenId order by created_timestamp desc limit 1)";
```
Comment (steven-sheehy) [inline hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/token/TokenAccessorImpl.java:88]: Where is the test that verifies that duplicate custom fees by it's JPA ID are returned?

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/token/TokenAccessorImpl.java [hunk 2]
```diff
@@ -89,7 +91,7 @@ public class TokenAccessorImpl implements TokenAccessor {
     private final TokenAccountRepository tokenAccountRepository;
     private final TokenAllowanceRepository tokenAllowanceRepository;
     private final NftAllowanceRepository nftAllowanceRepository;
-    private final CustomFeeRepository customFeeRepository;
+    private final CustomFeeRepositoryImpl customFeeRepositoryimpl;
```
Comment (steven-sheehy) [inline hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/token/TokenAccessorImpl.java:94]: Revert
```suggestion
    private final CustomFeeRepository customFeeRepository;
```

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Pull request merged**